widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -153,8 +153,12 @@ internal class ChatController(
             emitViewState { chatState.setLastTypedText("").setShowSendButton(isShowSendButtonUseCase("")) }
         }
 
-        override fun errorOperatorNotOnline(message: Unsent) {
-            onSendMessageOperatorOffline(message)
+        override fun onMessagePrepared(message: Unsent) {
+            appendUnsentMessage(message)
+        }
+
+        override fun errorOperatorNotOnline() {
```

Comment:
Just as an idea, maybe it will become a bit more readable if we rename `errorOperatorNotOnline()` to `errorOperatorOffline()`.

Reply:
changed